python/lsst/cp/pipe/defects.py

```diff
@@ -89,6 +89,12 @@ class MeasureDefectsTaskConfig(pipeBase.PipelineTaskConfig,
              "hot/bright pixels in dark images. Unused if thresholdType==``STDEV``."),
         default=5,
     )
+    biasThreshold = pexConfig.Field(
```

Comment:
I don't like that this does nothing in the `STDEV` case (meaning that entire task call is a NOOP), but I can't articulate why, so I think this is just a complaint with nothing to do.

Reply:
I don't love the way that VALUE/STDEV is smushed into one, and I refrained from doing anything more drastic to refactor this part because it could go on forever.  I'll add a warning at least if it's called that way.